app/routes/contact/__init__.py

```
@@ -23,109 +23,190 @@
+                        conversion_type="contact_message",
+                        user_ip="unknown",  # Will be set by middleware
+                        user_agent="unknown",  # Will be set by middleware
+                        metadata={
+                            "name": data.name,
+                            "email": data.email,
+                            "subject": data.subject,
+                            "company": data.company
+                        }
+                    )
+                    print("Contact message analytics conversion tracked successfully in database")
+                except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)